bin/p2-log-bridge/main.go

```diff
@@ -20,31 +21,56 @@ func main() {
 	kingpin.Version(version.VERSION)
 	kingpin.Parse()
 
+	if err := setSTDINToBlock(); err != nil {
+		logging.DefaultLogger.WithError(err).Error("fatal error setting STDIN to block")
+		os.Exit(1)
+	}
+
+	var wg sync.WaitGroup
 	loggerCmd := exec.Command(*durableLogger)
 	durablePipe, err := loggerCmd.StdinPipe()
 	if err != nil {
-		logging.DefaultLogger.WithError(err).Errorln("failed during logbridge setup")
-		os.Exit(1)
+		logging.DefaultLogger.WithError(err).Error("fatal error during configuration of subordinate log command")
 	}
 
-	go handleSignals(durablePipe)
+	wg.Add(1)
+	go func(loggerCmd exec.Cmd, wg sync.WaitGroup) {
```

Comment:
This goroutine needs a reference to the same `sync.WaitGroup` value that's defined in line 29, not its own copy.